rialto_airflow/harvest/authors.py

```diff
@@ -20,7 +20,7 @@ def load_authors_table(snapshot) -> str:
     logging.info(
         f"Loading authors from {authors_file} into database {snapshot.database_name}"
     )
-    with Session.begin() as session:
+    with Session.begin() as session:  # type: ignore
```

Comment:
instantiation of some sqlalchemy building block confuses the type checker again (a running theme 😆)

Reply:
Yes, maybe this will get easier once we can use sqlalchemy 2? 

https://docs.sqlalchemy.org/en/20/orm/extensions/mypy.html